activestate.macos.yaml

```diff
@@ -0,0 +1,70 @@
+constants:
+  - name: SYSTRAY_APP
+    if: eq .OS.Name "MacOS"
+    value: ActiveState\ Desktop.app
+scripts:
+  - name: build-systray-app
+    language: bash
+    standalone: true
+    if: eq .OS.Name "MacOS"
+    description: Copies the MacOS application directory to the build dir
+    value: |
+      mkdir -p build/${constants.SYSTRAY_APP}
+
+      cp -r installers/${constants.SYSTRAY_APP} build/
+  - name: generate-update
+    language: bash
+    standalone: true
+    if: eq .OS.Name "MacOS"
+    description: Generate update files
+    value: |
+      set -e
+      $constants.SET_ENV
+
+      export GOARCH=${1:-amd64}
+      mkdir -p ./build/update
+      go run scripts/update-generator/main.go --systemAppDir $BUILD_TARGET_DIR/$constants.SYSTRAY_APP -o ./build/update \
+      $BUILD_TARGET_DIR/$constants.BUILD_INSTALLER_TARGET \
+      $BUILD_TARGET_DIR/$constants.BUILD_DAEMON_TARGET \
+      $BUILD_TARGET_DIR/$constants.BUILD_SYSTRAY_TARGET \
+      $BUILD_TARGET_DIR/$constants.BUILD_TARGET \
+  - name: generate-test-update
```

Comment:
@mdrohmann this is making me increasingly uncomfortable. Can you please file a followup for cleaning up the testing of updates? This is way too much mocking.

Reply:
https://www.pivotaltracker.com/story/show/177957411

IMO this is still the best way to test the new update mechanism reliably, as it doesn't rely on "new" updates being available on S3 yet, which they are not (for the most part).  So, until the release channel has some of those updates, I would like to keep them.

I would also like to point out two advantages of these tests:
1. They test that `./scripts/update-generator` generates an installable update.  So, without these tests, we may be uploading some broken update and will only find out - after the fact.  That, of course, may not be so bad, as we should catch it on the beta-channel.
2. They make the tests much faster, as we do not have to update over the internet.

Reply:
I'm fine with us still doing some mocking. But if I broke these tests and I had to fix this looking at this code would make me scream.. This looks daunting. I just want to get it in a place where it doesn't make me do that :p